components/02-components/02-content-group/02-news-group/news-group.config.json

```diff
@@ -3,7 +3,9 @@
     "name": "news group",
     "status": "prototype",
 	"context": {
-		"text": "I am a news group."
+		"news-image": "/college-dls/utsa/images/News_Blog.png",
+        "news-image-01": "/college-dls/utsa/images/News_Blog-1.png",
+        "news-image-02": "/college-dls/utsa/images/News_Blog-2.png"
```

Comment:
Can we relocate these images to /college-dls/college/ instead of ../utsa/.  These will change from site to site, so shouldn't be part of /college-dls/utsa